src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/ClientConfigurationSettings.java

```diff
@@ -25,12 +25,27 @@ public class ClientConfigurationSettings {
     public static class ClientConfigurationCompletionSettings {
         /**
          * Whether or not completions should be offered as case-sensitive. Defaults to false.
-         **/
+         */
         public boolean caseSensitive = false;
     }
 
+    /**
+     * Client-side code workspace symbol settings.
+     */
+    public static class ClientConfigurationWorkspaceSymbolSettings {
```

Comment:
Client-side config additions. Again, I'm not married to these names, so feel free to suggest alternatives.